<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpimView</class>
 <widget class="QMainWindow" name="SpimView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpimView</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="xTextSegTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionRe_Load_Source_File"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Save_Log_FIle"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Simulator">
    <property name="title">
     <string>&amp;Simulator</string>
    </property>
    <addaction name="action_Clear_Registers"/>
    <addaction name="action_Reinitialize_Simulator"/>
    <addaction name="separator"/>
    <addaction name="action_Run"/>
    <addaction name="action_Single_Step"/>
    <addaction name="action_Multiple_Step"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Breakpoint"/>
    <addaction name="action_List_Breakpoints"/>
    <addaction name="separator"/>
    <addaction name="actionChange_Memory_Location"/>
    <addaction name="separator"/>
    <addaction name="action_Display_Symbols"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menu_Registers">
    <property name="title">
     <string>&amp;Registers</string>
    </property>
    <addaction name="actionBinary"/>
    <addaction name="actionOctal"/>
    <addaction name="actionHex"/>
    <addaction name="actionDecimal"/>
    <addaction name="actionChar"/>
   </widget>
   <widget class="QMenu" name="menu_Text_Segment">
    <property name="title">
     <string>&amp;Text Segment</string>
    </property>
    <addaction name="actionUser_Text"/>
    <addaction name="actionKernel_Text"/>
    <addaction name="separator"/>
    <addaction name="actionNarrow_Range"/>
    <addaction name="separator"/>
    <addaction name="actionComments"/>
    <addaction name="actionInstruction_Value"/>
   </widget>
   <widget class="QMenu" name="menu_Data_Segment">
    <property name="title">
     <string>&amp;Data Segment</string>
    </property>
    <addaction name="actionUser_Data"/>
    <addaction name="actionUser_Stack"/>
    <addaction name="actionKernel_Data"/>
    <addaction name="separator"/>
    <addaction name="actionNarrow_Display"/>
    <addaction name="separator"/>
    <addaction name="actionBinary_2"/>
    <addaction name="actionOctal_2"/>
    <addaction name="actionHex_2"/>
    <addaction name="actionDecimal_2"/>
    <addaction name="actionString"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionInteger_Registers"/>
    <addaction name="actionFP_Registers"/>
    <addaction name="actionText_Segment"/>
    <addaction name="actionData_Segment"/>
    <addaction name="actionConsole"/>
    <addaction name="actionSPIM_Messages"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Console"/>
    <addaction name="actionToolbar"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionView_Help"/>
    <addaction name="actionAbout_SPIM"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Simulator"/>
   <addaction name="menu_Registers"/>
   <addaction name="menu_Text_Segment"/>
   <addaction name="menu_Data_Segment"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="IntRegDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Integer Registers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QTextEdit" name="IntRegTextEdit">
    <property name="minimumSize">
     <size>
      <width>275</width>
      <height>310</height>
     </size>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="FPRegDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>FP Registers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QTextEdit" name="FPRegTextEdit">
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="TextSegDockWidget">
   <property name="windowTitle">
    <string>Text Segment</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QTextEdit" name="TextSegmentTextEdit">
    <property name="minimumSize">
     <size>
      <width>749</width>
      <height>600</height>
     </size>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="DataSegDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Data Segment</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QTextEdit" name="DataSegTextEdit">
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ConsoleDockWidget">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QTextEdit" name="ConsoleTextEdit">
    <property name="windowTitle">
     <string>Console</string>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open Source File</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close Source File</string>
   </property>
  </action>
  <action name="action_Save_Log_FIle">
   <property name="text">
    <string>&amp;Save Log FIle</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Clear_Registers">
   <property name="text">
    <string>&amp;Clear Registers</string>
   </property>
  </action>
  <action name="action_Reinitialize_Simulator">
   <property name="text">
    <string>Re&amp;initialize Simulator</string>
   </property>
  </action>
  <action name="actionRe_Load_Source_File">
   <property name="text">
    <string>Re&amp;Load Source File</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </action>
  <action name="action_Single_Step">
   <property name="text">
    <string>&amp;Single Step</string>
   </property>
  </action>
  <action name="action_Multiple_Step">
   <property name="text">
    <string>&amp;Multiple Step</string>
   </property>
  </action>
  <action name="actionSet_Breakpoint">
   <property name="text">
    <string>Set &amp;Breakpoint</string>
   </property>
  </action>
  <action name="action_List_Breakpoints">
   <property name="text">
    <string>&amp;List Breakpoints</string>
   </property>
  </action>
  <action name="actionChange_Memory_Location">
   <property name="text">
    <string>Change &amp;Memory Location</string>
   </property>
  </action>
  <action name="action_Display_Symbols">
   <property name="text">
    <string>&amp;Display Symbols</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionBinary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Binary</string>
   </property>
  </action>
  <action name="actionOctal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Octal</string>
   </property>
  </action>
  <action name="actionHex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </action>
  <action name="actionDecimal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Decimal</string>
   </property>
  </action>
  <action name="actionChar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Char</string>
   </property>
  </action>
  <action name="actionUser_Text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>User Text</string>
   </property>
  </action>
  <action name="actionKernel_Text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kernel Text</string>
   </property>
  </action>
  <action name="actionNarrow_Range">
   <property name="text">
    <string>Narrow Display</string>
   </property>
  </action>
  <action name="actionComments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </action>
  <action name="actionInstruction_Value">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Instruction Value</string>
   </property>
  </action>
  <action name="actionUser_Data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>User Data</string>
   </property>
  </action>
  <action name="actionUser_Stack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>User Stack</string>
   </property>
  </action>
  <action name="actionKernel_Data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kernel Data</string>
   </property>
  </action>
  <action name="actionNarrow_Display">
   <property name="text">
    <string>Narrow Display</string>
   </property>
  </action>
  <action name="actionBinary_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Binary</string>
   </property>
  </action>
  <action name="actionOctal_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Octal</string>
   </property>
  </action>
  <action name="actionHex_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </action>
  <action name="actionDecimal_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Decimal</string>
   </property>
  </action>
  <action name="actionString">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>String</string>
   </property>
  </action>
  <action name="actionInteger_Registers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Integer Registers</string>
   </property>
  </action>
  <action name="actionFP_Registers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FP Registers</string>
   </property>
  </action>
  <action name="actionText_Segment">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text Segment</string>
   </property>
  </action>
  <action name="actionData_Segment">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Data Segment</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actionSPIM_Messages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SPIM Messages</string>
   </property>
  </action>
  <action name="actionClear_Console">
   <property name="text">
    <string>Clear Console</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionStatus_Bar">
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionView_Help">
   <property name="text">
    <string>View Help</string>
   </property>
  </action>
  <action name="actionAbout_SPIM">
   <property name="text">
    <string>About SPIM</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
